net/Socket.cpp

```diff
@@ -1298,6 +1385,52 @@ LocalServerSocket::~LocalServerSocket()
 #  define LOG_CHUNK(X)
 #endif
 
+bool StreamSocket::checkForcedRemoval(std::chrono::steady_clock::time_point now) noexcept
```

Comment:
might need to move this impl to after 

"#endif // !MOBILEAPP" (if mobile needs it) and/or put ifdef MOBILEAPP around it in the header if it doesn't or a !MOBILE stub, something like that anyway

Reply:
yeah. Great eyes. Thx!